Jenkinsfile

```diff
@@ -6,12 +6,13 @@ pipeline{
   environment {
     MAVEN_HOME = "$WORKSPACE/.m2/"
     MAVEN_USER_HOME = "$MAVEN_HOME"
+    MAVEN_REPOSITORY = "$MAVEN_HOME/repository/"
```

Comment:
Generalize things so that we can just use MAVEN_USER_HOME instead of having to point to a /repository/ directory.